dissect/target/plugins/os/windows/task_helpers/tasks_xml.py

```diff
@@ -101,6 +101,8 @@ def __init__(self, task_element: Element, task_path: TargetPath):
         self.logon_type = self.get_element("Principals/Principal/LogonType")
         self.group_id = self.get_element("Principals/Principal/GroupId")
         self.display_name = self.get_element("Principals/Principal/DisplayName")
+        if self.display_name == None:
+            self.display_name = str(task_path)[str(task_path).rfind("/")+1:]
```

Comment:
```suggestion
        self.display_name = self.get_element("Principals/Principal/DisplayName") or task_path.name
```
I believe this achieves the same result.